### PR TITLE
Fix in  CCS Record Case

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
Added few missing fields
- `registered_status`
- `hh_caste`
- `disabled`

### CODE REVIEW

File: data_gen/data/randomizers.py [hunk 1]
```diff
@@ -163,3 +165,11 @@ def get_random_delivery_nature(randomObj):
 
 def get_still_live_cases(randomObj):
     return STILL_LIVE_CASES[randomObj.randint(0, len(STILL_LIVE_CASES) - 1)]
+
+
+def get_caste(randomObj):
+    return CASTES[randomObj.randint(0, len(STILL_LIVE_CASES) - 1)]
```
Comment (czue) [inline data_gen/data/randomizers.py]: I think this should use `random.choice`. and also that will fix the bug where you're referencing the length of the wrong variable. :wink:

File: data_gen/data/randomizers.py [hunk 2]
```diff
@@ -163,3 +165,11 @@ def get_random_delivery_nature(randomObj):
...
+
+
+def get_registered_status(randomObj):
+    return REGISTER_STATUS[randomObj.randint(0, len(REGISTER_STATUS) - 1)]
```
Comment (czue) [inline data_gen/data/randomizers.py]: `random.choice` seems better here too

File: data_gen/data/randomizers.py (hunk 1 above)
Comment (ShubhamBansal1997) [reply data_gen/data/randomizers.py]: done

File: data_gen/data/randomizers.py (hunk 2 above)
Comment (ShubhamBansal1997) [reply data_gen/data/randomizers.py]: done

File: data_gen/data/randomizers.py [hunk 3]
```diff
@@ -163,3 +164,11 @@ def get_random_delivery_nature(randomObj):
 
 def get_still_live_cases(randomObj):
     return STILL_LIVE_CASES[randomObj.randint(0, len(STILL_LIVE_CASES) - 1)]
+
+
+def get_caste(randomObj):
+    return CASTES[randomObj.randint(0, len(CASTES) - 1)]
```
Comment (czue) [inline data_gen/data/randomizers.py]: is there a reason you're still doing it this way instead of `random.choice`?

Comment (ShubhamBansal1997) [reply data_gen/data/randomizers.py]: Updated

**Pull request merged**